src/parser.js

```diff
@@ -3,13 +3,25 @@
 const path = require("path");
 const Handlebars = require("handlebars");
 const logger = require("winston-color");
-import { defaultKeyValueMap, initOptions, checkMapTypes, convertValue } from "./options.js";
+const common = require("commonmark/lib/common.js");
```

Comment:
Does commonmark export this? ( Is it a undocument API? )
We want to avoid to use direct require.